org/ystia/docker/containers/generic/types.yml

```diff
@@ -102,13 +102,16 @@ node_types:
         required: false
     attributes:
       container_id: { get_operation_output: [SELF, Standard, create, CONTAINER_ID] }
+      started_container_id: { get_operation_output: [SELF, Standard, create, STARTED_CONTAINER_ID] }
     interfaces:
       Standard:
         inputs:
           # Will be empty for create operation but not a big deal
           CONTAINER_ID: { get_attribute: [SELF, container_id] }
+          STARTED_CONTAINER_ID: { get_attribute: [SELF, started_container_id] }
           AUTO_REMOVE: { get_property: [SELF, auto_remove] }
           CLEANUP: { get_property: [SELF, cleanup] }
+          COMMAND: { get_property: [SELF, docker_run_cmd] }
```

Comment:
We should probably also deal with `docker_run_args`.
Note that the `docker_container` ansible module accept also a list of strings as a command...